frontend/src/restapi/players-profiles/PlayersProfilesRestApi.ts

```diff
@@ -1,13 +1,14 @@
 import axios from "axios";
 import { PlayerProfilesListDto } from "./PlayerProfilesListDto";
 import { PlayerProfileDto } from "./PlayerProfileDto";
+import { PATH_BASE_URL } from "../../components/atoms/constants/apiPaths";
```

Comment:
To chyba nie jest atoms? Ani nawet żaden komponent? Chyba dałbym to jak juz do rkatalogu estapi

Reply:
Widziałem na internecie, że takie rzeczy jak kolory, ścieżki czy inne stałe były w "atoms".